x-pack/plugins/ml/public/data_frame/components/job_create/job_create_form.tsx

```diff
@@ -52,6 +64,10 @@ export const JobCreateForm: SFC<Props> = React.memo(
 
     const [created, setCreated] = useState(defaults.created);
     const [started, setStarted] = useState(defaults.started);
+    const [indexPatternId, setIndexPatternId] = useState(defaults.indexPatternId);
+    const [progressPercentComplete, setprogressPercentComplete] = useState<undefined | number>(
```

Comment:
Should be `setProgressPercentComplete` (camelCase setProgress)

Reply:
Fixed in 199de93.